src/cf-core/src/cf-base.less

```diff
@@ -58,66 +63,74 @@
 //
 
 body {
-    .u-webfont-regular();
-
     color: @text;
+    font-family: 'Avenir Next', Arial, sans-serif;
     font-size: unit( @base-font-size-px / 16 * 100, % );
     line-height: @base-line-height;
 }
 
+button,
+input,
+select,
+textarea {
+    // Must set these explicitly to override Normalize.css's provided default:
+    // font-family: sans-serif;
+    font-family: 'Avenir Next', Arial, sans-serif;
+}
+
 .heading-1( @fs: @size-i ) {
     @font-size: @fs;
-    .u-webfont-regular();
 
     margin-bottom: unit( 15px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
+    font-weight: normal;
```

Comment:
Do we even need this if it's the default?

Reply:
…Or is this not the default here?

Reply:
The default weight (at the browser level) for headings is bold.